compiler_opt/es/blackbox_evaluator.py

```diff
@@ -97,12 +122,43 @@ def get_results(
       # update lists as work gets done
       _, not_done = concurrent.futures.wait(
           not_done, return_when=concurrent.futures.FIRST_COMPLETED)
-
     return futures
 
+  def get_results(
+      self, pool: FixedWorkerPool,
+      perturbations: list[bytes]) -> list[concurrent.futures.Future]:
+    # We should have _samples by now.
+    if not self._samples:
+      raise RuntimeError('Loaded samples are not available.')
+    return self._launch_compilation_workers(pool, perturbations)
+
   def set_baseline(self, pool: FixedWorkerPool) -> None:
-    del pool  # Unused.
-    pass
+    if self._baselines is not None:
```

Comment:
This function should probably get some test coverage?

Reply:
Yes. Coming up!